smart-gate/shell-scripts/SetNamespace.sh

```diff
@@ -0,0 +1,38 @@
+#Files
+ENV_FILE=".env"
+MUD_CONFIG_FILE="mud.config.ts"
+CONSTANTS_FILE="src/systems/constants.sol"
+
+#Colours
+GREEN="\033[32m"
+YELLOW="\033[33m"
+RESET="\033[0m"
+
+function validate_input(){
+    local INPUT=""
+    read -p "Please insert your $1: " INPUT
+    local MIN_LENGTH="$2"
```

Comment:
can we also add a max length as namespace is 14 bytes

Reply:
Good idea, thank you :) Have added